extension/src/popup/views/__tests__/SendTokenPayment.test.tsx

```diff
@@ -169,7 +177,7 @@ describe("SendTokenPayment", () => {
     </Wrapper>,
   );
 
-  it("can send a payment using a Soroban token", async () => {
+  it.skip("can send a payment using a Soroban token", async () => {
```

Comment:
looks like this test is dependent on a real testnet account and a real contract ID. We should probably mock these - otherwise we'll run into issues with this test every time there's a testnet reset, like today